docs/_basic/acknowledging_requests.md

```diff
@@ -13,7 +13,7 @@ We recommend calling `ack()` right away before sending a new message or fetching
 
 ```javascript
 // Regex to determine if this is a valid email
-let isEmail = /^[\w-\.]+@([\w-]+\.)+[\w-]{2,4}$/
+let isEmail = /^[\w\-\.]+@([\w\-]+\.)+[\w\-]+$/
```

Comment:
```suggestion
let isEmail = /^[\w\-\.\+]+@([\w\-]+\.)+[\w\-]+$/
```

Agh I can never copy and paste everything. Meant to include this `+` in the last update.

But this is just for the emails like `something+plus@example.com` - but it's a bit brutal. Regex is hard overall. I think this would accept too many `+` so your call on if this is worth including.